vstruct/tests/testbasic.py

```diff
@@ -151,7 +152,6 @@ def test_vstruct_varray(self):
         self.assertEqual( v[2], 0x41 )
 
     def test_bitfield(self):
-        from vstruct.bitfield import *
```

Comment:
Python 3 (at least in unit tests) forbids this. Although bitfield imports causes an `envi` import that isn't defined in setup.py/requirements in your vstruct fork.